src/vision/include/HSVFilterNode.h

```diff
@@ -24,7 +26,7 @@ class HSVFilterNode {
 
   public:
     HSVFilterNode(int argc, char** argv, std::string node_name);
-
+    // void mySigintHandler(int sig);
```

Comment:
hmm, this looks like a C++ style comment? Not actually sure what the effect will be, but we should probably just remove this line if we don't need it?

Reply:
Removed